src/components/FilterGroup.tsx

```diff
@@ -75,11 +79,21 @@ export function FilterGroup({
     };
   }, [customCssClasses]);
 
+  const replicateCollapsibleLabelCssClasses = useMemo(() => {
+    return twMerge('mb-4', builtInCollapsibleLabelCssClasses.label, cssClasses.titleLabel);
```

Comment:
Is it necessary to call twMerge here? I took a peak and CollapsibleLabel is using the classNames package instead of useComposedCssClasses (which uses twMerge). Is it possible to update CollapsibleLabel with useComposedCssClasses and then not have to import builtInCollapsibleLabelCssClasses here?

Reply:
`builtInCollapsibleLabelCssClasses` is needed because that is an export from CollapsibleLabel. This helps replicate the title labels style wether collapsible or not